site/content/usage/06-vpc-networking.md

```diff
@@ -160,3 +160,32 @@ See the complete example [here](https://github.com/weaveworks/eksctl/blob/master
 
 **Note**: Specifying the NAT Gateway is only supported during cluster creation and it is not touched during a cluster
 upgrade. There are plans to support changing between different modes on cluster update in the future.
+
+### Managing Access to the Kubernetes API Server Endpoints
+
+The default creation of an EKS cluster exposes the Kubernetes API server publicly but not directly
+from within the
+VPC subnets (Public=true, Private=false). Traffic destined for the API server from within the VPC
+must first exit
+the VPC networks (but not Amazon's network) and then re-enter to reach the API server. Allowing direct access from
+within the VPC subnets is possible but not without some caveats. See [Amazon EKS Cluster
+Endpoint Access Control](https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html) for
+further information
+on additional steps needed when public=false, private=true.
+
+The Kubernetes API server endpoint access for a cluster can be cofigured for public and private
+access by
+specifiying the `--vpc-endpoint-access-public` and/or`--vpc-endpoint-access-private` CLI flags to be
```

Comment:
Have you checked with @errordeveloper about adding these new flags? I think we are trying to minimize the amount of CLI flags we have so eksctl doesn't become too complicated.

Reply:
Didn't realize, I'll check with him.

Reply:
Yes, we are no adding new flags, especially in `eksctl create cluster`. We should only expose config fields for this, and sever flag-only users with whatever we pick as default.

Reply:
I'll remove the flags.